Editor/EditorCore/CutTool.cs

```diff
@@ -0,0 +1,1421 @@
+            if (currentEvent.type == EventType.KeyDown)
+                HandleKeyEvent(currentEvent);
+
+            if(currentEvent.type == EventType.Repaint)
+            {
+                DoExistingLinesGUI();
+                DoExistingPointsGUI();
+            }
+
+            if (EditorHandleUtility.SceneViewInUse(currentEvent))
+                return;
+
+            if(m_Mesh != null)
+            {
+                m_ControlId = GUIUtility.GetControlID(FocusType.Passive);
+                if(currentEvent.type == EventType.Layout)
+                    HandleUtility.AddDefaultControl(m_ControlId);
+
+                DoPointPlacement();
+
+                if(currentEvent.type == EventType.MouseMove)
+                    SceneView.RepaintAll();
+
+                if(currentEvent.type == EventType.Repaint)
+                {
+                    DrawGuideLine();
+                    DoCurrentPointsGUI();
+                    DoVisualCues();
+
+                    Cursor.SetCursor(m_CurrentCutCursor, Vector2.zero, CursorMode.Auto);
+                    if(m_CurrentCutCursor != null)
+                    {
+                        Rect sceneViewRect = window.position;
+                        sceneViewRect.x = 0;
+                        sceneViewRect.y = 0;
+                        SceneView.AddCursorRect(sceneViewRect, MouseCursor.CustomCursor);
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Overlay GUI
+        /// </summary>
+        /// <param name="target">the target of this overlay</param>
+        /// <param name="view">the current SceneView where to display the overlay</param>
+        void OnOverlayGUI(UObject target, SceneView view)
+        {
+            if(MeshSelection.selectedObjectCount != 1)
+            {
+                var rect = EditorGUILayout.GetControlRect(false, 45, GUILayout.Width(250));
+                EditorGUI.HelpBox(rect, L10n.Tr("One and only one ProBuilder mesh must be selected."), MessageType.Warning);
+            }
+
+            m_SnapToGeometry = DoOverlayToggle(L10n.Tr("Snap to existing edges and vertices"), m_SnapToGeometry);
+            EditorPrefs.SetBool(k_SnapToGeometryPrefKey, m_SnapToGeometry);
+
+            if(!m_SnapToGeometry)
+                GUI.enabled = false;
+            EditorGUI.indentLevel++;
+            using(new GUILayout.HorizontalScope())
+            {
+                EditorGUILayout.LabelField(L10n.Tr("Snapping distance"), GUILayout.Width(200));
+                m_SnappingDistance = EditorGUILayout.FloatField(m_SnappingDistance);
+                EditorPrefs.SetFloat( k_SnappingDistancePrefKey, m_SnappingDistance);
+            }
+            EditorGUI.indentLevel--;
+
+            GUI.enabled = true;
+
+            if(MeshSelection.selectedObjectCount != 1)
+                GUI.enabled = false;
+
+            if(m_Mesh == null)
+            {
+                if(GUILayout.Button("Start"))
```

Comment:
use `EditorGUIUtility.TrContent` for translation

Reply:
What is the difference between this method and L10n.Tr method?